test/hpr_protocol_router_SUITE.erl

```diff
@@ -31,7 +31,6 @@ all() ->
         grpc_test,
         grpc_connection_refused_test,
         grpc_full_flow_send_test,
-        grpc_full_flow_connection_refused_test,
```

Comment:
Why was this test removed?

Reply:
Hmm. I don't know why I would have touched that.
Merge problems?
I will take a look.

Reply:
Oh,  I remember now. 
The test appeared twice in the list. I removed a duplicate.